packages/frontend/src/components/svg/MyNearWalletLogo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const MyNearWalletLogo = ({ mode = '' }) => {
+const MyNearWalletLogo = ({ mode = Math.random().toString(32) }) => {
```

Comment:
just curious, what is this for?

Reply:
In this logo, <path> using filling with <linearGradient> like this

<path d="some.vectors" fill={`url(#myLinearGradient}`/>
<linearGradient id="myLinearGradient">

But when i use this svg several times at document, id becomes not uniq. This is a problem.
But Math.random().toString(32)  generates uniq hash for several svg instance.

For example, if i make some element with logo hidden, then and others logos reffering to first linearGradient becomes hidden.

I tried to replace linearGradient to css gradient, but unsuccefully